webserver/views/api.py

```diff
@@ -77,14 +78,15 @@ def submit_listen():
 @api_bp.route("/1/user/<user_id>/listens")
 def get_listens(user_id):
 
+    listenstore = listenstore_connection.get_listenstore()
```

Comment:
In the user profile method, the listenstore is created at the top, because it's used many times. I put this one here for symmetry, though perhaps you're right.